README.md

```diff
@@ -0,0 +1,182 @@
+# Liquid Crystal Library for Arduino
+
```

Comment:
I see some of the content is repeated on the website: https://www.arduino.cc/en/Reference/LiquidCrystal

I really think we should decide to have it in a single place, maybe link from the read me to the website? It makes things easier to maintain instead of have to update multiple places :)